src/app/FakeLib/GitVersionHelper.fs

```diff
@@ -0,0 +1,50 @@
+module Fake.GitVersionHelper
+
+open FSharp.Data
+open Newtonsoft.Json
+open System
+
+type GitVersionProperties = {
+                                Major : int;
+                                Minor : int;
+                                Patch : int;
+                                PreReleaseTag : string;
+                                PreReleaseTagWithDash : string;
+                                PreReleaseLabel : string;
+                                PreReleaseNumber : int;
+                                BuildMetaData : string;
+                                BuildMetaDataPadded : string;
+                                FullBuildMetaData : string;
+                                MajorMinorPatch : string;
+                                SemVer : string;
+                                LegacySemVer : string;
+                                LegacySemVerPadded : string;
+                                AssemblySemVer : string;
+                                FullSemVer : string;
+                                InformationalVersion : string;
+                                BranchName : string;
+                                Sha : string;
+                                NuGetVersionV2 : string;
+                                NuGetVersion : string;
+                                CommitsSinceVersionSource : int;
+                                CommitsSinceVersionSourcePadded : string;
+                                CommitDate : string;
+                            }
+
+
+let GitVersion () =
+    let dirs = [ (environVar "ProgramData")  @@ "chocolatey" @@ "bin" ]
+    let file = "GitVersion.exe"
+    let timespan =  TimeSpan.FromMinutes 1.
+    let usage = "To install GitVersion.exe, start PowerShell as Administrator and run choco install gitversion.portable -s https://chocolatey.org/api/v2"
```

Comment:
There's a [gitversion.commandline](https://www.nuget.org/packages/GitVersion.CommandLine) nuget package, so perhaps we should use the existing tools-path-finding that we do for other nuget-delivered exes? and then we've got conventions for executing those on mono already in FAKE.